GPU/GPUTracking/TRDTracking/GPUTRDTracker.cxx

```diff
@@ -48,6 +48,9 @@ class GPUTPCGMPolynomialField;
 
 #include "GPUChainTracking.h"
 
+template <class TRDTRK, class PROP>
+const float GPUTRDTracker_t<TRDTRK, PROP>::sRadialOffset = -0.1f;
```

Comment:
This will not work on the GPU, since this will be only available as a host variable.
You have to use CONSTEXPR.
For all the new code, we can just use the C++11 way (for non-array only), i.e. `static constexpr sRadialOffset = -0.1f`;
However, since the TRDTracker still runs in AliRoot, this does not work and you have to use CONSTEXPR with some tricks. See https://github.com/AliceO2Group/AliceO2/blob/dev/GPU/GPUTracking/SliceTracker/GPUTPCGeometry.h#L53.

Reply:
Do I also need to add the `#if !defined(__OPENCL__) || defined(__OPENCLCPP__)`? Wouldn't that mean that when compiled with OpenCL the tracking will not work?

Reply:
No, you don't. It will in fact not work with OpenCL1, but it will work with C++ For OpenCL (`__OPENCLCPP__`). Only the TPC slice tracker is compiled with OpenCL 1, for legacy reasons. So your code will never see `__OPENCL__` without `__OPENCLCPP__`.

Reply:
Thanks! For me the build was successful now and the CPU tracking works. Let's see about the CI..